master/internal/db/postgres_experiments_intg_test.go

```diff
@@ -430,16 +430,18 @@ func TestCheckpointMetadata(t *testing.T) {
 				m = &trialv1.TrialMetrics{
 					TrialId:        int32(tr.ID),
 					StepsCompleted: stepsCompleted,
-					Metrics: &structpb.Struct{
-						Fields: map[string]*structpb.Value{
-							defaultSearcherMetric: {
-								Kind: &structpb.Value_NumberValue{
-									NumberValue: metricValue,
+					Metrics: &checkpointv1.Metrics{
```

Comment:
minor nit, move `checkpointv1.Metrics` now?

Reply:
sorry, do what?

Reply:
i just dont know why `Metrics` are in `checkpointv1`.